#SRSFurnitureGeometry parent_obj_geometry
#SRSFurnitureGeometry[] furniture_geometry_list
srs_msgs/SRSSpatialInfo parent_obj_geometry
srs_msgs/SRSSpatialInfo[] furniture_geometry_list
---
geometry_msgs/Pose2D deliver_base_pose
float32 R



